apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -114,19 +123,32 @@ const CreateEventPage = ({ history }) => {
         .required('Event Description is required.')
         .max(5000, 'Event Description must be under 5000 characters.'),
       location: Yup.string().required('Event Location is required.'),
+      imageFile: Yup.mixed().required('Image is required.'),
     }),
 
-    onSubmit: (values) => {
+    onSubmit: async (values) => {
       notifySubmissionToast();
       if (!values.endDate) values.endDate = cloneDeep(values.startDate);
       addTimeToDate(values.startTime, values.startDate);
       addTimeToDate(values.endTime, values.endDate);
+      try {
+        var media = await uploadFile({ variables: { file: file } });
+      } catch (err) {
+        updateSubmissionToast(toast.TYPE.ERROR, 'Fatal Error: Event Not Created');
+        console.error(err);
+        return;
+      }
+      const imageMedia: IMedia = {
+        id: media.data.uploadFile.id,
+        mediaType: media.data.uploadFile.mediaType,
+      };
       const payload = {
```

Comment:
You can set the type of this to:
```ts
const payload: Partial<IEvent> = {
```
for IEvent, you can have a look at my example [here](https://github.com/RGPosadas/Mull/pull/80/files#diff-175bb5751a40c4aac4cf8ebea1c5678bb9c7a2f6101bfe447d88d5ade6915135R19-R30).